modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -2934,6 +2934,12 @@ export const console: ConsoleNS = {
                             message: "Step cannot be deleted"
                         }
                     },
+                    authenticationStepDeleteErrorDueToAppShared: {
+                        genericError: {
+                            description: "This authenticator is required for the shared application.",
```

Comment:
WDYT?

Since we are not showing the notification near the authenticator steps component, using the authenticator name in the description improves clarity IMO.

```suggestion
                            description: "The SSO authenticator is required for the shared application.",
```

Reply:
This is kept as a generic message. In case there are multiple authenticators in the future, that can not be deleted due to this "shared application" requirement.